test/CMakeLists.txt

```
@@ -406,3 +406,11 @@ add_subdirectory(ci)
 if(WIN32)
   add_subdirectory(performance)
 endif()
+
+add_custom_target(
+  check-package
+  COMMAND ${CMAKE_COMMAND}
+    -DCMAKE_PREFIX_PATH="${CMAKE_INSTALL_PREFIX}"
+    -B ${CMAKE_CURRENT_BINARY_DIR}/check-package-build
+    ${CMAKE_CURRENT_SOURCE_DIR}/packaging
```

```suggestion
  COMMAND ${CMAKE_COMMAND}
    -DCMAKE_PREFIX_PATH="${CMAKE_INSTALL_PREFIX};${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}"
    -B ${CMAKE_CURRENT_BINARY_DIR}/check-package-build
    ${CMAKE_CURRENT_SOURCE_DIR}/packaging
```

Source: https://github.com/microsoft/vcpkg/blob/ff6867374598312866fecfc64736a334591ceace/scripts/buildsystems/vcpkg.cmake#L420